app/containers/NewMediaCall/NewMediaCall.integration.test.tsx

```
@@ -0,0 +1,412 @@
+		//   CreateCall.handleCall → mediaSessionInstance.startCall('user-1', 'user')
+		//   → session.startCall('user', 'user-1') [args reversed for SDK]
+		//   → mock fires 'newCall' → MediaSessionInstance handler
+		//   → useCallStore.setCall + Navigation.navigate('CallView')
+		fireEvent.press(getByTestId('new-media-call-button'));
+
+		// SDK-boundary: args are (actor, userId) — reversed from the public API
+		expect(session.startCall).toHaveBeenCalledWith('user', 'user-1');
+		expect(Navigation.navigate).toHaveBeenCalledWith('CallView');
+		expect(mockHideActionSheet).toHaveBeenCalledTimes(1);
+
+		const call = useCallStore.getState().call;
```

ESLint

[error] Use object destructuring.

(prefer-destructuring)